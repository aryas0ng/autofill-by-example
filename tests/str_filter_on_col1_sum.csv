Anderson County,0,4
Anderson County,4,4
Bedford County,3,3
Benton County,8,23
Benton County,10,23
Benton County,5,23
Bradley County,4,
Bradley County,4,
Bradley County,21,
Bradley County,21,
Campbell County,10,
Cannon County,7,
Cannon County,4,
Coffee County,3,
Coffee County,3,
Coffee County,16,
Davidson County,16,
Davidson County,27,
Davidson County,8,
Greene County,3,
Greene County,3,
Hamilton County,11,
Hamilton County,4,