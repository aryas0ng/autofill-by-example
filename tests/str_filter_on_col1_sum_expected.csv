Anderson County,0,4
Anderson County,4,4
Bedford County,3,3
Benton County,8,23
Benton County,10,23
Benton County,5,23
Bradley County,4,50
Bradley County,4,50
Bradley County,21,50
Bradley County,21,50
Campbell County,10,10
Cannon County,7,11
Cannon County,4,11
Coffee County,3,22
Coffee County,3,22
Coffee County,16,22
Davidson County,16,51
Davidson County,27,51
Davidson County,8,51
Greene County,3,6
Greene County,3,6
Hamilton County,11,15
Hamilton County,4,15